mlir/include/mlir/Target/Cpp/CppEmitter.h

```diff
@@ -26,7 +26,8 @@ namespace emitc {
 /// arguments are declared at the beginning of the function.
 LogicalResult translateToCpp(Operation *op, raw_ostream &os,
                              bool declareVariablesAtTop = false,
-                             StringRef onlyTu = "");
+                             StringRef onlyTu = "",
+                             bool propagateConstants = false);
```

Comment:
For me `propagateConstants` sounds a bit misleading, since this is commonly used for the optimization which folds away constant expressions etc. (especially the test case name without the knowledge of this feature here could be misleading). How about `inlineConstants`?

Reply:
Another option would be `constantsAsVariables = true`, and setting it to false gives the new behavior.

Reply:
Yes, I like that 🙂

Reply:
Fixed.